sirepo/package_data/static/json/impactt-schema.json

```diff
@@ -186,31 +217,225 @@
+            "rf_frequency": ["RF Frequency [Hz]", "RPNValue", 0, "Frequency of the RF field in Hz."],
+            "rf_field_scale": ["RF field scale", "RPNValue", 1, "This can be used to scale the field amplitude. Normally set to 1."],
+            "solenoid_field_scale": ["Scale of solenoid B field", "RPNValue", 1],
+            "filename": ["Filename", "InputFile", "", "Solenoid field map filename"]
+        },
+        "SOLENOID": {
+            "name": ["NAME", "ValidatedString", "", ""],
+            "l": ["L [m]", "RPNValue", 0, "The element length"],
+            "radius": ["Radius [m]", "RPNValue", 0],
+            "filename": ["Filename", "InputFile", "", "Solenoid field map filename"]
         }
     },
     "view": {
+        "beamline": {
+            "title": "Beamline",
+            "advanced": [
+                "name",
+                "x",
+                "y",
+                "z",
+                "theta",
+                "phi",
+                "psi"
+            ]
+        },
+        "beamlineReport": {
+            "title": "Lattice",
+            "hasDataFile": false,
+            "advanced": []
+        },
+        "command_beam": {
+            "title": "Command Beam",
+            "description": "The BEAM statement defines data for the particles in a beam.",
+            "fieldsPerTab": 8,
+            "advanced": [
+                "n_p_row",
+                "n_p_col",
+                "d_t",
+                "n_step",
+                "n_bunch",
+                "dim",
+                "n_p",
+                "flag_err",
+                "flag_diag",
+                "flag_img",
+                "z_img",
+                "n_x",
+                "n_y",
+                "n_z",
+                "x_rad",
+                "y_rad",
+                "perd_len",
+                "flag_dist",
+                "n_emission",
+                "t_emission",
+                "b_curr",
+                "b_k_energy",
+                "b_mass",
+                "b_charge",
+                "b_freq",
+                "t_ini"
+            ]
+        },
         "simulationStatus": {
             "title": "Simulation Status",
             "advanced": []
+        },
+        "simulationSettings": {
+            "title": "Simulation Settings",
+            "basic": [],
+            "advanced": []
+        },
+        "BPM": {
+            "title": "BPM",
+            "description": "Beam Position Monitor",
+            "advanced": [
+                "name",
+                "l",
+                "bpmstp",
+                "bnseg"
+            ]
+        },
+        "DRIFTTUBE": {
+            "title": "DRIFTTUBE",
+            "description": "The DRIFTTUBE element defines a drift space.",
+            "fieldsPerTab": 8,
+            "advanced": [
+                "name"
+            ]
+        },
+        "DIPOLE": {
+            "title": "DIPOLE",
+            "description": "A Dipole",
```

Comment:
No need to have a "description" field here if it is just restating the name of the element displayed in the panel heading.